### DISCUSSION TITLE
个人设置的更新迁移

### DISCUSSION THREAD

**Discussion by Youxikong** [hooke007/mpv_PlayKit#201] [mpv 通用]
每次更新mpv lazy，因为我对设置有修改，都需要花大量时间重新设置。我觉得是不是可以在mpv.conf里面include一个设置文件，在里面填写需要覆盖默认设置的选项。

**Comment by hooke007** [comment]
- 当然是可以的，根据mpv.conf读取机制，最简单的就是把你要的选项偏好设置写在下方
https://github.com/hooke007/mpv_PlayKit/blob/19b0666a687fb5df10acc8049c4bb68921eed4c5/portable_config/mpv.conf#L5

- `--include` 也是个特殊的选项可以写多个而不会覆盖
https://github.com/hooke007/mpv_PlayKit/discussions/126

**Reply by Youxikong** [reply]
也就时说逐行生效，越往后优先级越高了。
我感觉还是include更方便一些，每次更新只需要改一行文字。

**Reply by Youxikong** [reply]
还有一个问题，input.conf有没有这种include的用法？我自己试着操作，也翻了文档，结果没有生效。

**Reply by hooke007** [reply]
https://github.com/hooke007/mpv_PlayKit/discussions/126#discussioncomment-3898523

--include 只能导入 "mpv.conf"
